### ISSUE TITLE
Is it better to use CPU and GPU together or use each separately?

### ISSUE THREAD

**Issue by dlsofree** [UnamSanctam/SilentCryptoMiner#2244]
Is it better to use CPU and GPU together or use each separately?
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/126412418/56459dc1-6944-4782-bec3-490e0e45a406)

**Comment by UnamSanctam** [comment]
Like you've done in the screenshot is the optimal way.

**Comment by dlsofree** [comment]
thanks [UnamSanctam](https://github.com/UnamSanctam) , I see that xmrig has released version 6.21, is it more optimal than version 6.19?

**Comment by UnamSanctam** [comment]
The current miner version is actually 6.20, I forgot to change the version file since it's in my excluded file list. There isn't too much of a difference between 6.20 and 6.21.1. At least not for anything that the silent miner (or Windows) uses.

**Comment by masterjek** [comment]
> Текущая версия майнера на самом деле 6.20, я забыл изменить файл версии, так как он находится в моем списке исключенных файлов. Между 6.20 и 6.21.1 особой разницы нет. По крайней мере, ни для чего, что использует тихий майнер (или Windows).

Are there any other updates planned for the silent miner in the future?

**Comment by UnamSanctam** [comment]
>Are there any other updates planned for the silent miner in the future?

Yes of course, like adding socks5 proxy (when connecting to a pool) support for both the CPU and GPU miners (that can be edited through "Remote Configuration" and the web panel). Sending some more information to the web panels new "Extra Data" field, such as RAM and other things. Adding more replacement strings (like how {COMPUTERNAME} works). And then of course reducing detections and more.
